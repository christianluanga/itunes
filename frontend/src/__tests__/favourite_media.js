import React from "react"
import FavouriteMedia from "../components/FavoriteMedia"
import renderer from "react-test-renderer"

it("Favourite Medias Renders Correctly", () => {
  const tree = renderer.create(<FavouriteMedia />).toJSON()
  expect(tree).toMatchSnapshot()
})
