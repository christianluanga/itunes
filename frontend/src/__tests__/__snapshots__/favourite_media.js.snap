// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favourite Medias Renders Correctly 1`] = `""`;
